runtimes/common/src/xcm_config.rs

```diff
@@ -146,23 +146,23 @@ parameter_types! {
 /// `AccountId`. This is used when determining ownership of accounts for asset
 /// transacting and when attempting to use XCM `Transact` in order to determine
 /// the dispatch Origin.
-pub type LocationToAccountId<RelayNetwork> = (
-	// The parent (Relay-chain) origin converts to the parent `AccountId`.
+pub type LocationToAccountId<NetworkId> = (
```

Comment:
The generic refers to a network ID. So especially for the future it is clearer to give the generic the proper name.